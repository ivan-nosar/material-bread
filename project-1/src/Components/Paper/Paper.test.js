import React from 'react';
import { Paper } from '../../';

import renderer from 'react-test-renderer';

test('Paper Renders', () => {
  const tree = renderer.create(<Paper />).toJSON();
  expect(tree).toMatchSnapshot();
});
