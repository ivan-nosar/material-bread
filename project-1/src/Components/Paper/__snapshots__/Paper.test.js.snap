// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paper Renders 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "borderRadius": 2,
      "shadowColor": "#000",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.23,
      "shadowRadius": 2.62,
    }
  }
/>
`;
